docs/user-guide.md

```diff
@@ -3,6 +3,16 @@
 * TOC
 {:toc}
 
+## Kedge Apply
+
+Deploy directly to Kubernetes, either updating or creating the artifacts if they don't exist.
+
+```sh
+$ kedge apply -f httpd.yaml
+deployment "httpd" created
+service "httpd" created
+```
+
 ## Kedge Create
 
 Deploy directly to Kubernetes without creating the artifacts. Internally, Kedge will generate the artifacts and then create it using the `kubectl` command.
```

Comment:
Should we mention that if artifacts already exist on a cluster this command will fail?

Reply:
I think it's self-explanatory as kubectl will fail with a descriptive error (artifact already exists).